pagopa/io-app
# fix: [IOBP-360] Fix dates format in ID Pay benificiary details screen

## Short description
This PR fixes the date format for last counter update date and onboarding date.

## List of changes proposed in this pull request
- [BeneficiaryDetailsContent.tsx](https://github.com/pagopa/io-app/compare/IOBP-360-fix-idpay-beneficiary-details-date-format?expand=1#diff-763f13f0b9f72e4763aa758664939937121a62cc637c86ff29bfe6f41ab7d56c): changed date format from `DD MMMM YYYY, hh:mm` to` DD MMM YYYY, HH:mm`

## How to test
Navigate to the beneficiary details screen and check that the dates are in the correct format.
